ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/Pruner.java

```diff
@@ -57,6 +58,7 @@ public Pruner(
   }
 
   public void start() {
+    LOG.info("Starting Pruner.");
```

Comment:
This should be at debug level.

Reply:
I was going along with this convention from the coding conventions:
```
* _Info_ 

  Generally useful information to log (for example, service start/stop, configuration assumptions). Default logging level
```
I thought it felt similar to other INFO logs at the beginning of a besu run and it would be good for a user to see it fast so that they may change it if that is not, in fact, what they wanted (hence why the coding conventions probably say this level is good for configuration assumptions). We could make a new INFO log at the configuration layer and set this one to DEBUG.

Reply:
If we do have this at info then it needs more info than "another loop is occurring" such as what the target horizon block is or possibly even the size of the DB (although that may be hard to get)

Reply:
For the end it would be useful to know how many blocks were pruned and how many were retained.

Reply:
I also have a higher bar for recurring info log items because we got feedback from infura we log too much stuff at info.

Reply:
Is this regarding the other logs? I agree that the recurring logs should be lowered to debug, I was making a case just for the pruner start log

Reply:
Is this logged every time the pruner does a loop or just once at startup.  If it's the latter then ok.

Reply:
Just once at startup, marking as resolved then